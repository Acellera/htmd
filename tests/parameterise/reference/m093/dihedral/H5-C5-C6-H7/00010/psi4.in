set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 0.612310	 0.826579	 -4.693528
C	 -0.328251	 1.300047	 -4.131739
C	 -0.494652	 1.149605	 -2.629745
C	 -1.378436	 2.095905	 -4.874098
C	 -0.832209	 -0.303459	 -2.258527
H	 -1.306527	 1.799607	 -2.322077
H	 0.406456	 1.465438	 -2.111787
C	 0.149006	 -1.302371	 -2.876593
H	 -0.830277	 -0.385436	 -1.177481
H	 -1.841814	 -0.541240	 -2.590045
C	 0.075323	 -1.399736	 -4.404711
H	 -0.023160	 -2.286784	 -2.445296
H	 1.161312	 -1.019151	 -2.595754
C	 -1.053427	 -2.288511	 -4.930067
H	 1.023541	 -1.787350	 -4.768483
H	 -0.024340	 -0.400645	 -4.824201
H	 -0.984148	 -3.291584	 -4.518035
H	 -1.004790	 -2.372263	 -6.011279
H	 -2.032469	 -1.894716	 -4.678683
H	 -2.325994	 1.564204	 -4.847814
H	 -1.075992	 2.239803	 -5.902012
H	 -1.538607	 3.059051	 -4.398180

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		6 3 2 1,
		6 3 5 8,
		3 5 8 12
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "22  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
