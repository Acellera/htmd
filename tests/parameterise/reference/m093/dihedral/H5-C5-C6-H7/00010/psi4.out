    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 17:20:02 2016

    Process ID:  20683
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 0.612310	 0.826579	 -4.693528
C	 -0.328251	 1.300047	 -4.131739
C	 -0.494652	 1.149605	 -2.629745
C	 -1.378436	 2.095905	 -4.874098
C	 -0.832209	 -0.303459	 -2.258527
H	 -1.306527	 1.799607	 -2.322077
H	 0.406456	 1.465438	 -2.111787
C	 0.149006	 -1.302371	 -2.876593
H	 -0.830277	 -0.385436	 -1.177481
H	 -1.841814	 -0.541240	 -2.590045
C	 0.075323	 -1.399736	 -4.404711
H	 -0.023160	 -2.286784	 -2.445296
H	 1.161312	 -1.019151	 -2.595754
C	 -1.053427	 -2.288511	 -4.930067
H	 1.023541	 -1.787350	 -4.768483
H	 -0.024340	 -0.400645	 -4.824201
H	 -0.984148	 -3.291584	 -4.518035
H	 -1.004790	 -2.372263	 -6.011279
H	 -2.032469	 -1.894716	 -4.678683
H	 -2.325994	 1.564204	 -4.847814
H	 -1.075992	 2.239803	 -5.902012
H	 -1.538607	 3.059051	 -4.398180

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		6 3 2 1,
		6 3 5 8,
		3 5 8 12
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "22  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-4-4.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 17:20:03 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           O          1.024529365516     0.823444856072    -0.820360647314    15.994914619560
           C          0.083968365516     1.296912856072    -0.258571647314    12.000000000000
           C         -0.082432634484     1.146470856072     1.243422352686    12.000000000000
           C         -0.966216634484     2.092770856072    -1.000930647314    12.000000000000
           C         -0.419989634484    -0.306593143928     1.614640352686    12.000000000000
           H         -0.894307634484     1.796472856072     1.551090352686     1.007825032070
           H          0.818675365516     1.462303856072     1.761380352686     1.007825032070
           C          0.561225365516    -1.305505143928     0.996574352686    12.000000000000
           H         -0.418057634484    -0.388570143928     2.695686352686     1.007825032070
           H         -1.429594634484    -0.544374143928     1.283122352686     1.007825032070
           C          0.487542365516    -1.402870143928    -0.531543647314    12.000000000000
           H          0.389059365516    -2.289918143928     1.427871352686     1.007825032070
           H          1.573531365516    -1.022285143928     1.277413352686     1.007825032070
           C         -0.641207634484    -2.291645143928    -1.056899647314    12.000000000000
           H          1.435760365516    -1.790484143928    -0.895315647314     1.007825032070
           H          0.387879365516    -0.403779143928    -0.951033647314     1.007825032070
           H         -0.571928634484    -3.294718143928    -0.644867647314     1.007825032070
           H         -0.592570634484    -2.375397143928    -2.138111647314     1.007825032070
           H         -1.620249634484    -1.897850143928    -0.805515647314     1.007825032070
           H         -1.913774634484     1.561069856072    -0.974646647314     1.007825032070
           H         -0.663772634484     2.236668856072    -2.028844647314     1.007825032070
           H         -1.126387634484     3.055916856072    -0.525012647314     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.08821  B =      0.05278  C =      0.04269 [cm^-1]
  Rotational constants: A =   2644.61803  B =   1582.36630  C =   1279.90355 [MHz]
  Nuclear repulsion =  407.104593956882297

  Charge       = 0
  Multiplicity = 1
  Electrons    = 64
  Nalpha       = 32
  Nbeta        = 32

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 76
    Number of basis function: 148
    Number of Cartesian functions: 148
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        148     148       0       0       0       0
   -------------------------------------------------------
    Total     148     148      32      32      32       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 318
    Number of basis function: 998
    Number of Cartesian functions: 998
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 3.5557378557E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -255.03810247715688   -2.55038e+02   7.07479e-02 
   @DF-RHF iter   2:  -205.06186209806327    4.99762e+01   6.02996e-02 DIIS
   @DF-RHF iter   3:  -296.08537670436255   -9.10235e+01   4.60145e-02 DIIS
   @DF-RHF iter   4:  -325.50387234961983   -2.94185e+01   3.31447e-02 DIIS
   @DF-RHF iter   5:  -338.56842140162263   -1.30645e+01   2.50799e-02 DIIS
   @DF-RHF iter   6:  -347.31633902958180   -8.74792e+00   8.15548e-03 DIIS
   @DF-RHF iter   7:  -347.91352239538190   -5.97183e-01   2.77015e-03 DIIS
   @DF-RHF iter   8:  -347.99674754351810   -8.32251e-02   9.82522e-04 DIIS
   @DF-RHF iter   9:  -348.01057014459599   -1.38226e-02   4.52304e-04 DIIS
   @DF-RHF iter  10:  -348.01323182883124   -2.66168e-03   1.38357e-04 DIIS
   @DF-RHF iter  11:  -348.01357024008013   -3.38411e-04   3.38574e-05 DIIS
   @DF-RHF iter  12:  -348.01359430847816   -2.40684e-05   9.23753e-06 DIIS
   @DF-RHF iter  13:  -348.01359685532856   -2.54685e-06   4.63818e-06 DIIS
   @DF-RHF iter  14:  -348.01359744297048   -5.87642e-07   1.69044e-06 DIIS
   @DF-RHF iter  15:  -348.01359757314373   -1.30173e-07   5.53846e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.560414     2A    -11.334497     3A    -11.236689  
       4A    -11.234834     5A    -11.227255     6A    -11.203504  
       7A    -11.189559     8A    -11.165791     9A     -1.428415  
      10A     -1.089323    11A     -1.041083    12A     -0.996311  
      13A     -0.928538    14A     -0.849240    15A     -0.793455  
      16A     -0.737295    17A     -0.674009    18A     -0.656537  
      19A     -0.639220    20A     -0.607709    21A     -0.594125  
      22A     -0.575883    23A     -0.567619    24A     -0.551025  
      25A     -0.535893    26A     -0.526858    27A     -0.501822  
      28A     -0.484804    29A     -0.447829    30A     -0.440636  
      31A     -0.407925    32A     -0.371972  

    Virtual:                                                              

      33A      0.139747    34A      0.214004    35A      0.259110  
      36A      0.269859    37A      0.275920    38A      0.284042  
      39A      0.300015    40A      0.313678    41A      0.319346  
      42A      0.333590    43A      0.340178    44A      0.355468  
      45A      0.364721    46A      0.379595    47A      0.390477  
      48A      0.395555    49A      0.419858    50A      0.447986  
      51A      0.454887    52A      0.481017    53A      0.498495  
      54A      0.553143    55A      0.704646    56A      0.729174  
      57A      0.741695    58A      0.745293    59A      0.771954  
      60A      0.792535    61A      0.792685    62A      0.818180  
      63A      0.833903    64A      0.882988    65A      0.907392  
      66A      0.909094    67A      0.926995    68A      0.965361  
      69A      0.991880    70A      1.010619    71A      1.015474  
      72A      1.050362    73A      1.070468    74A      1.105757  
      75A      1.114243    76A      1.142772    77A      1.154491  
      78A      1.161313    79A      1.176862    80A      1.183561  
      81A      1.190128    82A      1.207065    83A      1.210965  
      84A      1.226491    85A      1.229567    86A      1.256141  
      87A      1.264418    88A      1.283777    89A      1.289776  
      90A      1.324008    91A      1.388660    92A      1.398400  
      93A      1.438831    94A      1.476650    95A      1.574582  
      96A      1.635473    97A      1.677098    98A      1.686326  
      99A      1.756640   100A      1.789049   101A      1.839721  
     102A      1.893621   103A      1.913481   104A      1.998500  
     105A      2.026085   106A      2.069969   107A      2.088238  
     108A      2.122804   109A      2.140488   110A      2.186206  
     111A      2.203258   112A      2.228551   113A      2.263332  
     114A      2.271301   115A      2.298476   116A      2.375289  
     117A      2.381123   118A      2.416178   119A      2.428104  
     120A      2.464028   121A      2.511262   122A      2.522676  
     123A      2.578122   124A      2.659688   125A      2.671596  
     126A      2.681118   127A      2.700614   128A      2.732271  
     129A      2.742816   130A      2.808087   131A      2.849838  
     132A      2.875634   133A      2.949184   134A      2.987977  
     135A      3.006422   136A      3.093182   137A      3.101206  
     138A      3.137747   139A      3.368766   140A      3.398636  
     141A      4.497948   142A      4.554015   143A      4.615916  
     144A      4.689742   145A      4.814275   146A      4.909976  
     147A      5.015276   148A      5.048924  

    Final Occupation by Irrep:
              A 
    DOCC [    32 ]

  Energy converged.

  @DF-RHF Final Energy:  -348.01359757314373

   => Energetics <=

    Nuclear Repulsion Energy =            407.1045939568822973
    One-Electron Energy =               -1279.0362777241903132
    Two-Electron Energy =                 523.9180861941640615
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -348.0135975731438975

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -4.3316      Y:    -3.6474      Z:     0.9647

  Electronic Dipole Moment: (a.u.)
     X:     3.5540      Y:     4.4432      Z:    -0.4286

  Dipole Moment: (a.u.)
     X:    -0.7775      Y:     0.7957      Z:     0.5361     Total:     1.2350

  Dipole Moment: (Debye)
     X:    -1.9763      Y:     2.0225      Z:     1.3626     Total:     3.1389


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-4-4.cx1.hpc.ic.ac.uk at Tue Jul  5 17:20:21 2016
Module time:
	user time   =      17.75 seconds =       0.30 minutes
	system time =       0.17 seconds =       0.00 minutes
	total time  =         18 seconds =       0.30 minutes
Total time:
	user time   =      17.75 seconds =       0.30 minutes
	system time =       0.17 seconds =       0.00 minutes
	total time  =         18 seconds =       0.30 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -4.3316      Y:    -3.6474      Z:     0.9647

  Electronic Dipole Moment: (a.u.)
     X:     3.5540      Y:     4.4432      Z:    -0.4286

  Dipole Moment: (a.u.)
     X:    -0.7775      Y:     0.7957      Z:     0.5361     Total:     1.2350

  Dipole Moment: (Debye)
     X:    -1.9763      Y:     2.0225      Z:     1.3626     Total:     3.1389

  Quadrupole Moment: (Debye Ang)
    XX:   -53.7310     YY:   -50.7191     ZZ:   -51.9730
    XY:    -2.6457     XZ:     2.1185     YZ:    -0.1441

  Traceless Quadrupole Moment: (Debye Ang)
    XX:    -1.5900     YY:     1.4219     ZZ:     0.1681
    XY:    -2.6457     XZ:     2.1185     YZ:    -0.1441

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     O     4.26236  4.26236  0.00000 -0.52473
       2     C     2.74688  2.74688  0.00000  0.50625
       3     C     3.19531  3.19531  0.00000 -0.39063
       4     C     3.28826  3.28826  0.00000 -0.57651
       5     C     3.16761  3.16761  0.00000 -0.33522
       6     H     0.41476  0.41476  0.00000  0.17048
       7     H     0.39775  0.39775  0.00000  0.20450
       8     C     3.15498  3.15498  0.00000 -0.30996
       9     H     0.41258  0.41258  0.00000  0.17484
      10     H     0.41879  0.41879  0.00000  0.16242
      11     C     3.17051  3.17051  0.00000 -0.34101
      12     H     0.42287  0.42287  0.00000  0.15427
      13     H     0.41912  0.41912  0.00000  0.16177
      14     C     3.23849  3.23849  0.00000 -0.47697
      15     H     0.42561  0.42561  0.00000  0.14878
      16     H     0.37068  0.37068  0.00000  0.25863
      17     H     0.42921  0.42921  0.00000  0.14158
      18     H     0.42332  0.42332  0.00000  0.15335
      19     H     0.43506  0.43506  0.00000  0.12987
      20     H     0.40521  0.40521  0.00000  0.18957
      21     H     0.39569  0.39569  0.00000  0.20862
      22     H     0.40496  0.40496  0.00000  0.19008

   Total alpha = 32.00000, Total beta = 32.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
